tests/test_core.py

```diff
@@ -6506,6 +6566,8 @@ def test_zzz_cases(self):
       shortname = os.path.splitext(name)[0]
       # TODO: test only worked in non-fastcomp (well, these cases)
       basename = os.path.basename(shortname)
+      if basename == returns_non_zero[-1]:
```

Comment:
what does this line do? seems like it's checking `basename == "zeroextarg"` (last element in the array), but that seems weird?

Reply:
Oops, didn't mean to commit that.. reverting.